api/management/api.go

```diff
@@ -225,10 +225,11 @@ type BackOfficeConfiguration map[string]map[string][]string
 // RealmAdminConfiguration struct
 type RealmAdminConfiguration struct {
 	Mode                *string                   `json:"mode"`
-	AvailableChecks     map[string]bool           `json:"available-checks"`
+	AvailableChecks     map[string]bool           `json:"available_checks"`
```

Comment:
If wew want to change this, we need to synchronize with abilis as they are now using this attribute

Reply:
And sync with fratt for the BO